native/CMakeLists.txt

```diff
@@ -1472,6 +1472,12 @@ cocos_source_files(
                  cocos/renderer/pipeline/custom/details/SerializationUtils.h
                  cocos/renderer/pipeline/custom/details/Set.h
                  cocos/renderer/pipeline/custom/details/Utility.h
+                 cocos/renderer/pipeline/profile/GPUTimeQuery.cpp
+                 cocos/renderer/pipeline/profile/GPUTimeQuery.h
+                 cocos/renderer/pipeline/profile/GPUStatisticsQuery.cpp
+                 cocos/renderer/pipeline/profile/GPUStatisticsQuery.h
+                 cocos/renderer/pipeline/profile/PipelineProfiler.cpp
+                 cocos/renderer/pipeline/profile/PipelineProfiler.h
```

Comment:
Might rename folder `profile` to `profiler`. Profile as noun has different meaning.

Reply:
Yes, this should be the `profiler`.